sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -163,3 +163,21 @@ export class ApplicationSummaryAPIOutDTO {
   submitted?: Date;
   status: ApplicationStatus;
 }
+
+/**
+ * History of SIN validations associated with a user.
+ */
+export class SINValidationsAPIOutDTO {
+  id: number;
+  sin: string;
+  createdAt: Date;
+  isValidSIN?: boolean;
+  sinStatus?: string;
+  validSINCheck?: string;
+  validBirthdateCheck?: string;
```

Comment:
Please change to validBirthDateCheck?

Reply:
I can rename but Birthdate is an acceptable way to write birthdate 😉 
https://www.thefreedictionary.com/birthdate
![image](https://user-images.githubusercontent.com/61259237/175357198-144cd9b8-e640-4012-be0a-925b00ec8a5a.png)

Reply:
This one is also following the DB migration creation previously as `valid_birthdate_check`. So to keep it consistent with DB it would be good to keep it.